ibis/backends/base/sqlglot/compiler.py

```diff
@@ -362,7 +362,7 @@ def visit_DefaultLiteral(self, op, *, value, dtype):
             if math.isnan(value):
                 return self.NAN
             elif math.isinf(value):
-                return self.POS_INF if value < 0 else self.NEG_INF
+                return self.POS_INF if value > 0 else self.NEG_INF
```

Comment:
Whoops :sweat_smile:

Reply:
Strange that it wasn't captures elsewhere.